docs/source/release/v6.1.0/mantidworkbench.rst

```diff
@@ -23,7 +23,8 @@ Added Floating/On Top setting for all the windows that are opened by workbench (
   to find better estimates of parameter errors.
 
 - Instrument view: when in tube selection mode, the sum of pixel counts is now output to the selection pane.
-
+- LoadLog algorithm will detect old unsupported log files and display a log message to inform the user.
```

Comment:
Tiny thing. Could we move this to framework notes? It's purely non-gui so would appear on the command-line Python script so belongs with the framework changes really.